lib/evss/disability_compensation_form/form0781.rb

```diff
@@ -19,11 +19,27 @@ def translate
         @final_output['veteranPhone'] = @phone_email['primaryPhone']
         @final_output['veteranSecondaryPhone'] = '' # No secondary phone available in 526 PreFill
         @final_output['veteranServiceNumber'] = '' # No veteran service number available in 526 PreFill
+
+        # The pdf creation functionality is looking for a single street address
+        # instead of a hash
+        @final_output['incidents'].map do |incident|
```

Comment:
minor, does this need use `.map` rather than `.each` since it's not returning an array?

Reply:
You are correct, I'll change it